next.config.js

```diff
@@ -17,5 +17,14 @@ module.exports = withTM({
         permanent: true,
       },
     ]
-  }
-});
+  },
+  webpack5: true,
```

Comment:
@nenadV91 what was the motivation of the change? 
Is this something that should use some comment?